### PR TITLE
Add missing signatures description for NMA calls

### PR DESCRIPTION
**Pull request by devgiants**
Add all NMA signature methods in order to provide all possible choices regarding user needs.

### CODE REVIEW

File: bundles/action/org.openhab.action.nma/README.md [hunk 1]
```diff
@@ -4,8 +4,11 @@ Sends push messages to your Android devices.
 
 ## Actions
 
-- `notifyMyAndroid(String event, String description)`: Send a message to a the pre configured api key (account) and use the configured or default values for the other parameters
+- `notifyMyAndroid(String event, String description)`: Send a message to a the pre-configured api key (account) and use the configured or default values for the other parameters. Envent is the notification title, and description the details.
```
Comment (watou) [inline bundles/action/org.openhab.action.nma/README.md]: Event

File: bundles/action/org.openhab.action.nma/README.md [hunk 2]
```diff
@@ -4,8 +4,11 @@ Sends push messages to your Android devices.
 
 ## Actions
 
-- `notifyMyAndroid(String event, String description)`: Send a message to a the pre configured api key (account) and use the configured or default values for the other parameters
+- `notifyMyAndroid(String event, String description)`: Send a message to a the pre-configured api key (account) and use the configured or default values for the other parameters. Envent is the notification title, and description the details.
+- `notifyMyAndroid(String event, String description, int priority)`: Send a message to a the pre-configured api key (account) with priority level (-2 to 2, from very low to emergency) and use the configured or default values for the other parameters.
```
Comment (watou) [inline bundles/action/org.openhab.action.nma/README.md]: "to the"

**Comment by watou** [comment]
Thanks for this!  Two small typos I noticed.

**Comment by devgiants** [comment]
Sorry for those typos, it's now fixed.
Thanks again for your work !

**Comment by watou** [comment]
Thank you for fixing the doc!  It ought to appear at docs.openhab.org after the next time the site content is regenerated (an occasional manual procedure).

**Pull request merged**